common/proto/corfu_features.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
+
+package org.corfudb.common.util.proto;
+
+enum Feature {
+  FEATURE_INVALID = 0;
```

Comment:
Can we not have an INVALID that way each feature just represents a bit position starting with index 0?
First feature = 0 - > takes first bit position at index 0
Second feature = 1 -> takes second bit at index 1

Reply:
I was following the practice - "Always Start Enums with INVALID = 0;" for backward compatibility. I will send you the links for further information on this.